initial version

FabricioFFC opened the pull request:
What includes:

* find path in locales files, where path can be a string ("home") or a string as a path ("home.title")
* returns error when path is not found in any locale file
* accepts localesPath and translateFunctionName as params

Commit:
initial version

What includes:

* find path in locales files, where path can be a string ("home") or a string as a path ("home.title")
* returns error when path is not found in any locale file
* accepts localesPath and translateFunctionName as params

commited by Fabrício Ferrari de Campos at Vizir sponsored by Natura

Commit:
chore: add commit message template

darthaud commented on index.js:
```diff
@@ -0,0 +1,61 @@
+const pathLocalizator = require('./src/path-localizator');
```
I think is better use pathLocator instead of pathLocalizator.

darthaud commented on src/path-localizator.js:
```diff
@@ -0,0 +1,45 @@
+};
+
+module.exports = function pathLocalizator(path, locales) {
+  const translations = getTranslations(locales);
```
Maybe you can execute getTranslations method on create to improve performance

darthaud commented on index.js:
```diff
@@ -0,0 +1,61 @@
+const pathLocalizator = require('./src/path-localizator');
+
+const runRule = (path, locales, context, node) => {
+  const result = pathLocalizator(path, locales);
```
Maybe you can stop execution of runRule when path is empty.

Commit:
docs: update README

Commit:
chore: update package.json info

Commit:
chore: add issue and pull request templates

Commit:
refactor: improve performance and code flow

Improvements based on the code review: https://github.com/natura-cosmeticos/eslint-plugin-i18n-checker/pull/1#pullrequestreview-168535866

FabricioFFC commented:
@darthaud thanks! I solved the issues that you reported: https://github.com/natura-cosmeticos/eslint-plugin-i18n-checker/pull/1/commits/64a4464eef0fedc01f70836f03fe739e20aa956b

Commit:
chore: add editorconfig and fix lint errors

darthaud commented:
Looks good to me.

darthaud reviewed (approved):
Looks good to me
